src/Integration.Vsix/CFamily/CFamilyHelper.cs

```diff
@@ -117,60 +117,56 @@ private static Request CreateRequest(FileConfig fileConfig, string absoluteFileP
             return defaults;
         }
 
-        internal /* for testing */ static Response CallClangAnalyzer(Request request, IProcessRunner runner, ILogger logger, CancellationToken cancellationToken)
+        internal /* for testing */ static void CallClangAnalyzer(Action<Message> handleMessage, Request request, IProcessRunner runner, ILogger logger, CancellationToken cancellationToken)
         {
-            string tempFileName = null;
             try
             {
-                tempFileName = Path.GetTempFileName();
-
-                // Create a FileInfo object to set the file's attributes
-                FileInfo fileInfo = new FileInfo(tempFileName);
-
-                // Set the Attribute property of this file to Temporary. 
-                // Although this is not completely necessary, the .NET Framework is able 
-                // to optimize the use of Temporary files by keeping them cached in memory.
-                fileInfo.Attributes = FileAttributes.Temporary;
-
-                using (var writeStream = new FileStream(tempFileName, FileMode.Open))
+                if (analyzerExeFilePath == null)
```

Comment:
Minor: no need for this check being inside the try-catch block.

It seems odd that we're checking for null here, but presumably there's a reason the code was like that before.